controller/model/session_handlers.go

```diff
@@ -325,6 +327,47 @@ func (handler *SessionHandler) ListSessionsForEdgeRouter(edgeRouterId string) (*
 	return result, nil
 }
 
+func (handler *SessionHandler) OnApiSessionDelete(name string, data []byte) {
+	apiSessionId := string(data)
+
+	logger := pfxlog.Logger().WithFields(map[string]interface{}{
+		"event":        name,
+		"apiSessionId": apiSessionId,
+	})
+
```

Comment:
Can we have the scan and delete be in separate transactions, please? Also, you have it in a loop but don't have a limit on the scan.

Reply:
Oh, is this not used anymore, replaced by the method in ApiSessionStore? If that's the case, can probably delete this

Reply:
This isn't used anymore. This is the old version and I never deleted it.

Reply:
Removed.